crates/wasmtime/src/engine.rs

```diff
@@ -239,6 +239,22 @@ impl Engine {
         Ok(mmap.to_vec())
     }
 
+    /// Returns a value that can be used to check precompiled WebAssembly compatibility.
+    ///
+    /// The outputs of [`Engine::precompile_module`] and [`Engine::precompile_component`]
+    /// are compatible with a different [`Engine`] instance only if the two engines use
+    /// compatible [`Config`]s. The value returned by this function will be the same for two
+    /// [`Engine`]s if a binary from one can be deserialized by the other.
+    #[cfg(compiler)]
+    #[cfg_attr(nightlydoc, doc(cfg(feature = "cranelift")))] // see build.rs
```

Comment:
Unclear to me if this is needed for this method.

Reply:
If this compiles without the `#[cfg]` no need for the attribute. If the `#[cfg]` is required, though, then this attribute is also required for better display in documentation.